dev/RadioButtons/TestUI/RadioButtonsPage.xaml

```diff
@@ -186,10 +186,10 @@
                         <TextBlock Text="RadioButtons Messages &amp; Events" Foreground="Red" HorizontalAlignment="Center"/>
 
                         <StackPanel Orientation="Horizontal" Grid.Row="1">
-                            <Button x:Name="ClearRadioButtonsEventsButton" AutomationProperties.Name="ClearRadioButtonsEventsButton" Content="Clear" Click="ClearRadioButtonsEventsButton_Click" Margin="1"/>
+                            <Button x:Name="ClearRadioButtonsEventsButton" AutomationProperties.Name="ClearRadioButtonsEventsButton" Content="Clear" Click="ClearRadioButtonsEventsButton_Click" Margin="1" IsTabStop="False"/>
```

Comment:
In my added interaction test, I set focus inside the RadioButtons group and select the next item by pressing keyboard down arrow key:
https://github.com/microsoft/microsoft-ui-xaml/pull/3137/files#diff-ba4a1a53f42aae082c207f00ed0bbce8R822

Before I do that, however, I clear the logged event messages by invoking (or clicking) the ClearRadioButtonsEventsButton. This takes away keyboard focus from my RadioButtons group and as such the test fails. Thus, I removed this button from the tab flow to not "steal" keyboard focus.

Is there are better way to do this?

I could, after clearing the logs, set focus to the RadioButtons group again, then verify focus us set, and finally do the keyboard selection check (instead of disallowing tab stop for the button).

Reply:
Hmm I see. I think clearing logs, setting focus again and then changing selection again is fine too given.

Would there be a way to test this without resetting the event log? After all, the event should show up in the log.

Reply:
> Would there be a way to test this without resetting the event log? After all, the event should show up in the log.

Yes, I don't actually need to clear the event log output. However, I think it is easier to understand the test by always clearing the log before I do another selection which will raise the SelectionChanged event. I.e. in case the event was raised one time too much, I could end up with a Verify output message similar to:
```
Expected <4>. Actually <5>.
```
The currrent output would read in this case:
```
Expected <1>. Actually <2>.
```
I think that's easier to work with here, hence why I always clear the logs after each SelectionChange raising operation.

Reply:
That's a good point, though we could also point out with comments that the event should have been raised "X times until Y" instead of trying to assert that it always has been raised once since last clear. But I don't think this makes a big difference. It's just a bit strange to remove something from the tab order that generally is included.

While this app does not get shipped, we still shouldn't make it inaccessible without a special reasons.

Reply:
I can check if manually setting back focus will work (as it should) and if it does, add this as an extra preparation step prior to the keyboard selection test and restore the button to be part of the tab flow.

Reply:
That sounds good.

Reply:
Done.